src/test/resources/application.yml

```diff
@@ -114,6 +117,8 @@ akhq:
         attributes:
           topics-filter-regexp:
             - "test.*"
+          acls-filter-regexp:
```

Comment:
I am not sure this is necessary because this group does not have `acls/read` role

Reply:
@LittleWat - Same comment as one below

Reply:
Thanks!